### PR TITLE
MAT-9098 displaying owner name on the cql library details page

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-9098](https://jira.cms.gov/browse/MAT-9098)
(Optional) Related Tickets:

### Summary
displaying owner name on the cql library details page

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editCqlLibrary/EditCqlLibrary.tsx [hunk 1]
```diff
@@ -68,6 +70,7 @@ const EditCqlLibrary = () => {
   const navigate = useNavigate();
   const { search } = useLocation();
   const values = queryString.parse(search);
+  const userServiceApi = useUserServiceApi();
```
Comment (gregory-akins) [inline src/components/editCqlLibrary/EditCqlLibrary.tsx]: Should we be consistent about using a useRef for the API calls?

**Review by gregory-akins — APPROVED** [review]
